packages/web-components/examples/codesandbox/usage/v18/index.html

```diff
@@ -0,0 +1,97 @@
+    <link rel="shortcut icon" href="https://www.ibm.com/favicon.ico" />
+
+    <title>@carbon/ibmdotcom-web-components example with v18</title>
+
+    <!-- Initialize Northstar -->
+    <script>
+      digitalData = { page: { pageInfo: { ibm: {} } } };
+    </script>
+    <script src="https://1.www.s81c.com/common/stats/ida_stats.js"></script>
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/v18/css/www.css" />
+    <script src="https://1.www.s81c.com/common/v18/js/www.js"></script>
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/v18/css/grid-fluid.css"/>
+
+    <!-- Configurable Northstar options -->
+    <script type="text/javascript">
+      IBMCore.common.util.config.set({
+        // configurations go here
+      });
+    </script>
+
+    <!-- Stylesheet to override Northstar -->
+    <style type="text/css">
+      #ibm-content {
+        padding-top: calc(3rem + 1px);
+      }
+    </style>
+
+    <!-- Application module that imports Carbon for IBM.com Web Components masthead module -->
+    <script type="module">
+      import '@carbon/ibmdotcom-web-components/es/components/masthead/masthead-container.js';
+    </script>
+  </head>
+  <body id="ibm-com" class="ibm-type">
+    <div id="ibm-top" class="ibm-landing-page">
+      <dds-masthead-container></dds-masthead-container>
+      <div id="ibm-content-wrapper">
+        <!-- HEADER WITH L1 NAVIGATION, LEADSPACE, TAB NAVIGATION -->
+
+        <main role="main" aria-labelledby="ibm-pagetitle-h1">
+          <div id="ibm-pcon">
+            <div id="ibm-content">
+              <div id="ibm-content-body">
+                <div id="ibm-content-main">
+                  <!-- CONTENT & COMPONENTS START -->
+                  <div class="ibm-fluid ibm-padding-bottom-3">
+                    <div class="ibm-col-12-4">
+                      <h1 class="ibm-h1 ibm-large ibm-light">
+                        Northstar v18 Starter Kit
+                      </h1>
+                      <p class="ibm-textcolor-gray-50">
+                        Page content and other components go here
+                      </p>
+                    </div>
+                  </div>
+                  <!-- CONTENT & COMPONENTS END -->
+                </div>
+              </div>
+            </div>
+          </div>
+        </main>
+      </div>
+
+      <!-- FOOTER_BEGIN -->
+      <footer role="contentinfo" aria-label="IBM">
+        <div id="ibm-footer">
+          <h2 class="ibm-access">Footer links</h2>
+          <ul>
+            <li><a href="https://www.ibm.com/contact/us/en/">Contact</a></li>
+            <li><a href="https://www.ibm.com/privacy/us/en/">Privacy</a></li>
+            <li><a href="https://www.ibm.com/legal/us/en/">Terms of use</a></li>
+            <li>
+              <a href="https://www.ibm.com/accessibility/us/en/"
+                >Accessibility</a
+              >
+            </li>
+          </ul>
+        </div>
+      </footer>
```

Comment:
Wonder if we should include `dds-footer-container` here instead too?

Reply:
Good question @jeffchew - Wanted to demonstrate major Carbon for IBM.com component and major Northstar component can co-exist. That's why the footer is of Northstar. Please let me know if you have any better idea to demonstrate such co-existence. Thanks!

Reply:
Ok, since it's an example I think it should be enough!